mw-webapp/src/component/header/Header.tsx

```diff
@@ -36,7 +40,6 @@ export const Header = () => {
         !
       </h2>}
       {/* //TODO: we need delete this navigate after main features about dayReports table will be finished */}
-      {user && <Navigate to={pages.user.path(user.uid)} />}
       <div className={styles.blockButton}>
```

Comment:
if you deleted code need to actualize comments too. Or delete it if it's no need more

Reply:
done

Reply:
delete